index.html

```diff
@@ -348,12 +348,12 @@
                         <thead><tr><th>Device Button Changes</th><th><code>button</code></th></tr></thead>
                         <tbody>
                             <tr><td>Neither buttons nor touch/pen contact changed since last event</td><td>-1</td></tr>
-                            <tr><td>Left Mouse,<br>Touch Contact,<br>Pen contact (with no modifier buttons pressed)</td><td>0</td></tr>
+                            <tr><td>Left Mouse,<br>Touch Contact,<br>Pen contact</td><td>0</td></tr>
```

Comment:
related, see https://github.com/w3c/pointerevents/issues/97 - this may be a quirk of Wacom Intuos Pro and its driver. Perhaps it's best to leave this out for now, under the implied assumption that in this case the pen/driver will internally treat the hovering click button press as a "contact", which means it's covered adequately already by having `Pen contact` in the text there